norm_script_alpha.py

```diff
@@ -0,0 +1,42 @@
+from fuzzywuzzy import process
+
+def get_normalized(el: str, li: list):
+    reading = process.extractBests(el, li, score_cutoff=75)
+    candidates = [el]
+    best_n = el
+    max_n = len(reading)
+
+    while reading != []:
+        (new,points) = reading.pop()
+        if points < 100 and new not in candidates:
+            candidates.append(new)
+            tmp = process.extractBests(new, li, score_cutoff=75)
+            reading = reading + tmp
+            if len(tmp) > max_n:
+                best_n = new
+    return (best_n, candidates)
+
+
+def write_csv(out: dict):
+    f = open("n_alpha.csv", "w")
+    f.write("Old;Normalized\n")
+    for k in out.keys():
+        for n in out[k]:
+            f.write(n+';'+k+'\n')
```

Comment:
Please use fstrings to improve code readability.
```suggestion
            f.write(f"{n};{k}\n"')
```